### DISCUSSION TITLE
Configuration file in subdirectories does not recognize the content

### DISCUSSION THREAD

**Discussion by ghost** [tailwindlabs/tailwindcss#9843] [Help] [answered]
**Reproduce the problem:**
```
npm install tailwindcss
```
Independent subdirectories:
```
mkdir config
mkdir templates
```
Tailwind config & input file
```
touch config/tailwind.config.js
touch config/index.css
```
Standalone Templates:
```
touch templates/index.html
touch templates/another.html # not in use for this test
```
config/tailwind.config.js (config for `templates/index.html` as example):
```
/** @type {import('tailwindcss').Config} */
module.exports = {
  content: ['./templates/index.html'],
  theme: {
    extend: {},
  },
  plugins: [],
}
```
config/index.css:
```
@tailwind base;
@tailwind components;
@tailwind utilities;
```
And if you now execute Tailwind:
```
npx tailwindcss -c config/tailwind.config.js -i config/index.css -o style.css
```
Do I get this message:
```
Rebuilding...

warn - No utility classes were detected in your source files. If this is unexpected, double-check the `content` option in your Tailwind CSS configuration.
warn - https://tailwindcss.com/docs/content-configuration

Done in 200ms.
```
The documentary states:
```
Paths are relative to your project root, not your tailwind.config.js file, so if your tailwind.config.js file is in a custom location, you should still write your paths relative to the root of your project.
```
But where is the root directory if the configuration is in a subdirectory? And why does it not recognize the TailwindCSS classes in this configuration (templates/index.html)? The background: I want to create a standalone configuration for each HTML page as specified.

**Comment by wongjn** [comment, marked as answer]
As the warning message states:

> No utility classes were detected in your source files

The warning should disappear if you add some text to `templates/index.html` that matches any Tailwind class, such as `block`, for example.

**Comment by ghost** [comment]
It works. Because if you start TailwindCSS in the root directory and put the classes inside given content file.
